### PR TITLE
Improve fetchmail

### LINKED ISSUE #711: Fetchmail only syncs the INBOX folder
I enabled the fetchmail feature to migrate from an old mail-in-a-box setup, but it seems to have only pulled in the INBOX items. All the other IMAP folders are still present in the source mailbox and missing from my Mailu mailbox. I am using 1.5 so I don't have the feature to disable "remove mails" yet; The source INBOX is empty, but the other folders are still there indicating they haven't even been seen?

### LINKED ISSUE #1231: E-Mails fetched by fetchmail trigger enough rspamd rules to be marked as spam
I currently fetch some emails from GMail which get filtered through rspamd and due to the nature of how the headers look almost all of them get marked as spam or straight up rejected. Here is a list of applied rules directly related to fetchmail for some common message:
```
VIOLATED_DIRECT_SPF (3.5)
HFILTER_HELO_5 (3)
GOOGLE_FORWARDING_MID_MISSING (2.5)
FORGED_RECIPIENTS (2)
R_SPF_FAIL (1)
BAD_REP_POLICIES (0.1)
RCVD_NO_TLS_LAST (0.1)
```
which adds up to 12.5. With the default mark as spam being 15 anything else will set it off.

### LINKED ISSUE #2246: Fetchmail: Missing support for '*_ADDRESS' env vars
### Environment
`docker-compose`

### Versions
`1.9`

## Description
According to the [documentation](https://mailu.io/1.9/configuration.html#infrastructure-settings) `*_ADDRESS` variables can be set to allow for DNS based container resolution. This works fine for all Mailu containers except for `fetchmail`, where only the following variables are evaluated in `fetchmail.py`:

- `HOST_ADMIN`
- `HOST_SMTP`

As a workaround one can set `HOST_ADMIN` and `HOST_SMTP` manually on the `fetchmail` container to get the same behaviour, but it would be great to use the existing `mailu.env` for `fetchmail` too.

## Expected behaviour
`fetchmail` honors `ADMIN_ADDRESS` and `SMTP_ADDRESS`, when set in `mailu.env`.

### PR DESCRIPTION
**Pull request by nextgens**
## What type of PR?

enhancement

## What does this PR do?

Improve fetchmail:
- allow delivery via LMTP (faster, bypassing the filters)
- allow several folders to be retrieved
- run fetchmail as non-root
- tweak the compose file to ensure we have all the dependencies

### Related issue(s)
- closes #1231 
- closes #2246 
- closes #711

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [ ] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

**Comment by nextgens** [comment]
bors try

**Comment by nextgens** [comment]
bors try

**Comment by nextgens** [comment]
bors try

**Comment by nextgens** [comment]
bors try

File: docs/webadministration.rst [hunk 1]
```diff
@@ -157,7 +157,11 @@ You can add a fetched account by clicking on the `Add an account` button on the
 
 * Keep emails on the server. When ticked, retains the email message in the email account after retrieving it.
 
-Click the submit button to apply settings. With the default polling interval, fetchmail will start polling the email account after 10 minutes.
+* Scan emails. When ticked, all the fetched emails will go through the local filters (rspamd, clamav, ...).
+
+* Folders. A comma separated list of folders to fetch from the server.
```
Comment (Diman0) [inline docs/webadministration.rst:162]: Please add that this is optional. That when it is not configured it will pull the INBOX folder.

**Review by Diman0 — CHANGES_REQUESTED** [review]
See in-line comment.

On a first glance the code looks fine. I will first run a couple of tests.

File: docs/webadministration.rst (hunk 1 above)
Comment (nextgens) [reply docs/webadministration.rst]: done in 15b889fa

File: core/admin/mailu/ui/forms.py [hunk 2]
```diff
@@ -41,6 +41,15 @@ def __call__(self, form, field):
         if not pattern.match(field.data.replace(" ", "")):
             raise validators.ValidationError(self.message)
 
+class MultipleFoldersVerify(object):
```
Comment (Diman0) [inline core/admin/mailu/ui/forms.py:44]: Please add a docstring that explains what this class checks. In other words explain what the regular expression verifies.

**Review by Diman0 — APPROVED** [review]
See in-line comment.
On the fetched accounts page I also notice that the column Folders is empty. I've tested with folders value
- blank (so INBOX is used as default value)
- INBOX
- INBOX, Junk

Is it expected for this column to be blank?
image

File: core/admin/mailu/ui/forms.py (hunk 2 above)
Comment (nextgens) [reply core/admin/mailu/ui/forms.py:44]: Fixed in c79e8d3852aac0c57f88bb2a930ffe68e761403e ... this has been introduced in 6474108

**Comment by szelga** [comment]
did this pull request made into the 1.9 branch?

**Comment by nextgens** [comment]
No, it hasn't been backported. It will be part of next release which is due any day now.

**Comment by szelga** [comment]
thanks.

**Pull request merged**